playlist_collection/m3u_to_playlist.py

```diff
@@ -58,11 +67,12 @@ def m3u_to_playlist(ssn, library_name: str, file_path: str, users: list=['@me'])
 	# Setup arg parsing
 	parser = ArgumentParser(description='Convert a .m3u file to a plex playlist')
 	parser.add_argument('-l','--LibraryName', type=str, help='Name of target library', required=True)
+	parser.add_argument('-p','--LibraryPath', type=str, help='Path within Plex to the Music files. Make sure it ends with /.', default='', required=False)
```

Comment:
It doesn't have to end with a `/`. Could also end with a `\` if the server is on Windows...